packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -99,5 +102,12 @@ export default class CardAdoptionChain extends Component<Signature> {
     }
 
     this.cardInheritanceChain = cardInheritanceChain;
+    this.args.setTotalFields?.(
```

Comment:
Consider moving `this.cardInheritanceChain` one level up in the component hierarchy so that you can pass data down (not up). You could make the total count a getter so that it recomputes when the data in inheritance chain changes. You have to expect it will change–remember we will add adding and removing fields in the schema editor.

Reply:
Updated [b5ebe79](https://github.com/cardstack/boxel/pull/701/commits/b5ebe79590e5ecb1f0ef0360f78be7ee77748783)